certified-connectors/1Me Corporate/apiDefinition.swagger.json

```diff
@@ -0,0 +1,179 @@
+  "schemes": [
+    "https"
+  ],
+  "consumes": [],
+  "produces": [],
+  "paths": {
+    "/api/CardTempletes/": {
+      "get": {
+        "responses": {
+          "default": {
+            "description": "default",
+            "schema": {}
+          }
+        },
+        "operationId": "RetrieveCardTemplates",
+        "summary": "Retrieve all corporate card templates",
+        "description": "Retrieve all corporate card templates",
+        "parameters": []
+      }
+    },
+    "/api/Invitation/": {
+      "post": {
+        "responses": {
+          "default": {
+            "description": "default",
+            "schema": {}
+          }
+        },
+        "summary": "Create invitation and send email for your corporate members",
+        "operationId": "SendInvitation",
+        "description": "Create invitation and send email for your corporate members",
+        "parameters": [
+          {
+            "name": "body",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "type": "object",
+              "properties": {
+                "CardTemplateId": {
+                  "type": "string",
+                  "description": "Card Template Id",
+                  "title": "Card Template Id"
+                },
+                "NameOnCard": {
+                  "type": "string",
+                  "description": "Name On Card",
+                  "title": "Name On Card"
+                },
+                "Jobtitle": {
+                  "type": "string",
+                  "description": "Job Title",
+                  "title": "Job Title"
+                },
+                "WorkEmail": {
+                  "type": "string",
+                  "description": "Work Email",
+                  "title": "Work Email"
+                },
+                "Extension": {
+                  "type": "string",
+                  "description": "Extension",
+                  "title": "Extension"
+                }
+              },
+              "required": [
+                "CardTemplateId",
+                "Jobtitle",
+                "WorkEmail"
+              ]
+            }
+          }
+        ]
+      }
+    },
+    "/api/Invitation/Disassociate": {
+      "delete": {
+        "responses": {
+          "default": {
+            "description": "default",
+            "schema": {}
+          }
+        },
+        "operationId": "DisassociateMember",
+        "summary": "Disassociate members from your corporate",
+        "description": "Disassociate members from your corporate",
+        "parameters": [
+          {
+            "description":"Disassociate members from your corporate",
+            "x-ms-summary":"Disassociate members from your corporate",
+            "name": "Content-Type",
+            "in": "header",
+            "required": true,
+            "type": "string",
+            "default": "application/json"
+          },
+          {
+            
```

Comment:
@MICHAEL-MAURICE all files should be properly formatted. please remove all empty lines from the swagger file.